api/internal/handler/data_loader/route_import.go

```diff
@@ -137,6 +137,22 @@ func (h *ImportHandler) Import(c droplet.Context) (interface{}, error) {
 		}
 	}
 
+	// merge route
+	idRoute := make(map[string]*entity.Route)
+	for _, route := range routes {
+		if existRoute, ok := idRoute[route.RouteId]; ok {
+			uris := append(existRoute.Uris, route.Uris...)
+			existRoute.Uris = uris
+		} else {
+			idRoute[route.RouteId] = route
```

Comment:
It seems that the combination of `method` is not considered.

Reply:
`methods` may already merged by `OpenAPI3ToRoute` method via path, and route have different path become different routes.
Here I just merge path via original route id.